src/main/java/com/amihaiemil/docker/RtLogs.java

```diff
@@ -65,6 +60,16 @@
      * Base URI.
      */
     private final URI baseUri;
+
+    /**
+     * Flag for stdout option.
+     */
+    private final boolean stdout;
+
+    /**
+     * Flag for stderr option.
+     */
+    private final boolean stderr;
```

Comment:
@bkuzmic Can you leave a puzzle so we find a more elegant solution than having them as class attributes? There are more params for logs, so maybe it should be a ``Map`` or something.